
public class Exception2 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub

		try {
			int a=100;
			int b=0;
			int result =a/b;
			
			System.out.println(result);
		}catch(ArithmeticException e) {
		
			System.out.println("It is  my pen");
		}
	}

}
